cts-java/src/test/java/org/ga4gh/cts/api/sequenceAnnotations/FeaturesSearchIT.java

```diff
@@ -81,20 +81,18 @@ public void checkExpectedNumberOfFeatures() throws InvalidProtocolBufferExceptio
     public void checkFeaturesSearchByParentId() throws InvalidProtocolBufferException, UnirestException, GAWrapperException {
         final long start = 0;
         final long end = 100000000;
-        final int expectedNumberOfFeatures = 50;
+        final int expectedNumberOfFeatures = 14;
```

Comment:
42

In case there's any doubt, here's the SQL query:
```
sqlite> select * from feature where type="transcript" limit 1;
4384884304|4384886160|[4385435792,4385439440,4384882896,4384884432,4385435920,4386169168,4384884112,4386169232,4386169296,4386169360,4386169424,4386169488,4386169552,4386169616,4386169680,4386170000,4386169744,4386169808,4384883536,4386169872,4385435856,4386169936,4384883856,4386170064,4384883920,4386170128,4386170192,4386170768,4386170256,4386170320,4385439312,4386170384,4386170448,4386169104,4385439376,4386170576,4385439504,4386170704,4386170512,4385439632,4385439568,4385439696]|ref_brca1|ENSEMBL|transcript|2|79821|.|-|ENST00000351666.3|BRCA1|BRCA1-203|{"gene_name":["BRCA1"],"gene_status":["KNOWN"],"havana_gene":["OTTHUMG00000157426.5"],"Parent":["ENSG00000012048.15"],"level":["3"],"transcript_status":["KNOWN"],"gene_id":["ENSG00000012048.15"],"protein_id":["ENSP00000338007.3"],"tag":["basic"],"gene_type":["protein_coding"],"transcript_id":["ENST00000351666.3"],"transcript_name":["BRCA1-203"],"transcript_type":["protein_coding"],"ID":["ENST00000351666.3"]}
sqlite> select count(*) from feature where parent_id=4384884304;
42
```